test/test_misc.F90

```diff
@@ -46,6 +50,20 @@ program test_misc
   if (lun .ne. 1 .or. il .ne. -1 .or. im .ne. 0) stop 4
   call closbf(11)
 
+  ! Test ireadmm()
+  open(unit = 11, file = 'testfiles/IN_9', form = 'UNFORMATTED', iostat = ios)
+  if (ios .ne. 0) stop 3
+  call ufbmem(11, 0, iret, iunit)
+  imsg = 1
+  if (ireadmm(imsg, subset, idate) .ne. 0) stop 4
+  if (imsg .ne. 2 .or. subset .ne. 'ADPSFC' .or. idate .ne. 23022519) stop 5
+  call closbf(11)
+
+  ! Test iupb().
+  mbay(1) = 1
+  mbay(2) = 2
+  if (iupb(mbay, 1, 1) .ne. 0) stop 6
+  
```

Comment:
Similar to my above comment for intest1.F90, this test_misc.F90 already has `stop 3`, `stop 4`, and `stop 5` lines in the code, and the idea is for stop numbers to be unique in a test code, correct?

Reply:
OK